grammar.js

```diff
@@ -798,8 +801,9 @@ const rules = {
       opt($.attribute_modifier),
       opt($._class_modifier),
       opt($._class_modifier),
+      opt('xhp'),
```

Comment:
Can we make this a named rule like `$.final_modifier`. We don't need to add it to `$._class_modifier` but I think it makes sense that this rule show up in the concrete syntax tree like `final` and `abstract` keywords do. 

Seems useful to have it appear as rule, other wise (depending on the particular tool using this tree-sitter parser/grammar) will have to read source code to know whether the class is an xhp class or not.
https://github.com/antoniodejesusochoasolano/tree-sitter-hack/blob/e381df2aa9847e51f1f3b9f0c9c175ea617cf1fe/grammar.js#L971-L975